src/app/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input } from '@angular/core';
+import { TechRecord } from '../models/tech-record.model';
+
+// largely copied from  archive/../vehicle-summary.component.ts
+@Component({
+  selector: 'app-vehicle-technical-record',
+  templateUrl: './vehicle-technical-record.component.html',
+  styleUrls: ['./vehicle-technical-record.component.scss']
+})
+export class VehicleTechnicalRecordComponent{
+  @Input() technicalRecord?: TechRecord;
+
+  constructor() { }
+
+  get vehicleClassDescription(): string | undefined {
+    return this.technicalRecord?.vehicleClass.description;
+  }
+
+  get isStandardVehicle(): boolean {
+    return (
+      'hgv' === this.technicalRecord?.vehicleType ||
+      'trl' === this.technicalRecord?.vehicleType ||
+      'psv' === this.technicalRecord?.vehicleType);
+  }
+
+  axlesHasParkingBrakeMrk(): boolean {
+    return (
+      this.technicalRecord != undefined &&
```

Comment:
Do we know if this logic is what we want?

Reply:
Removed as per your earlier comment of not having started this area of work yet